src/rdkafka_admin.c

```diff
@@ -4509,6 +4617,7 @@ rd_kafka_DeleteAclsResponse_parse(rd_kafka_op_t *rko_req,
                                   char *errstr,
                                   size_t errstr_size) {
         const int log_decode_errors = LOG_ERR;
+        rd_kafka_broker_t *rkb      = rko_req->rko_rk->rk_internal_rkb;
```

Comment:
What's this for? we'll want to reference the reply rkb, if any.

Reply:
Hi, it's for the `rd_rkb_log` . I've tried `rko_req->rko_replyq.q->rkq_rk->rk_internal_rkb` also now, but gives segmentation fault. Is it the correct way to access it?

Reply:
ok, I've found it's like in the other response parse functions